iib/workers/tasks/build.py

```diff
@@ -670,20 +677,23 @@ def handle_add_request(
 
     set_request_state(request_id, 'in_progress', 'Creating the manifest list')
     output_pull_spec = _create_and_push_manifest_list(request_id, arches)
-
-    if legacy_support_packages:
-        export_legacy_packages(
-            legacy_support_packages, request_id, output_pull_spec, cnr_token, organization
-        )
-
-    _finish_request_post_build(
+    modified_index_image = _update_index_image_pull_spec(
         output_pull_spec,
         request_id,
         arches,
         from_index,
         overwrite_from_index,
         overwrite_from_index_token,
     )
+    if legacy_support_packages:
+        export_legacy_packages(
+            legacy_support_packages, request_id, modified_index_image, cnr_token, organization
+        )
+    set_request_state(
+        request_id,
+        'complete',
+        'Operator bundle(s) added to index image, completing the request successfully',
```

Comment:
Same as my comment in the rm endpoint.
```suggestion
        'The operator bundle(s) were successfully added to the index image',
```

Reply:
I added it to consider another feedback assuming "the request completed successfully" was being more meaningful

Reply:
I'm sorry, but I don't understand what you mean. Could you please elaborate or rephrase your comment?

Reply:
In one of my earlier patches, the message was 'Operator bundle(s) added to index image', but later to consider the suggested feedback that 'The request has been completed successfully' should be added,  I changed it 'Operator bundle(s) added to index image, completing the request successfully' and looks like adding it this way isn't any better
 I like this suggestion made by you and I think it's better than both the earlier messages I used, so I will add 'The operator bundle(s) were successfully added to the index image' in next patch.

Reply:
This is the feedback Tulsi is talking about. https://github.com/release-engineering/iib/pull/76#discussion_r425423401